smrf/framework/CoreConfig.ini

```diff
@@ -580,293 +620,432 @@ wind_ninja_height:            default = 5.0,
-                  type = float,
-                  description = Threshold for drift cells measured in degrees from tbreak file
-
-min_scour:       default = 0.55,
-                 type = float,
-                 description = minimum multiplier for precip redistribution to account for wind scour
-
-max_scour:       default = 1.0,
-                 type = float,
-                 description = max multiplier for precip redistribution to account for wind scour
-
-min_drift:       default = 1.0,
-                 type = float,
-                 description = min multiplier for precip redistribution in a drift cell
-
-max_drift:       default = 3.5,
-                 type = float,
-                 description = max multiplier for precip redistribution in a drift cell
-
-# polynomial coefficients
-drift_poly_a:    default = 0.0289,
-                 type = float,
-                 description = first coefficient for drift factor function
-
-drift_poly_b:    default = -0.0956,
-                 type = float,
-                 description = second coefficient for drift factor function
-
-drift_poly_c:    default = 1.000761,
-                 type = float,
-                 description = third coefficient for drift factor function
-
-ppt_poly_a:      default = 0.0001737,
-                 type = float,
-                 description = firstcoefficient for scour factor function
-
-ppt_poly_b:      default = 0.002549,
-                 type = float,
-                 description = second coefficient for scour factor function
-
-ppt_poly_c:      default = 0.03265,
-                 type = float,
-                 description = thirdcoefficient for scour factor function
-
-ppt_poly_d:      default = 0.5929,
-                 type = float,
-                 description = coefficient for scour factor function
+type = bool,
+description = Use local elevation gradients in gridded interpolation
+
+grid_local_n:
+default = 25,
+type = int,
+description = number of closest grid cells to use for calculating elevation gradient
+
+idw_power:
+default = 2.0,
+type = float,
+description = Power for decay of a stations influence in inverse distance weighting
+
+dk_ncores:
+default = 2,
+type = int,
+description = Number of threads to use in the dk calculation
+
+storm_mass_threshold:
+default = 1.0,
+type = float,
+description = Start criteria for a storm in mm of measured precipitation in
+              millimeters in any pixel over the domain.
+
+# TODO CHANGE TO HOURS INSTEAD OF TIMESTEPS #99
+marks2017_timesteps_to_end_storms:
+default = 6,
+type = int,
+description = number of timesteps to elapse with precip under start criteria
+              before ending a storm.
+
+susong1999_timesteps_to_end_storms:
+default = 6,
+type = int,
+description = number of timesteps to elapse with precip under start criteria
+              before ending a storm.
+
+new_snow_density_model:
+default = susong1999,
+options = [marks2017 susong1999 piecewise_susong1999],
```

Comment:
Also not sure. The question is whether we want to delete code thats being deprecated. For sure it should be removed from the core config but do we want to leave code available from deprecated bits? I am thinking not...

Reply:
I left this in so we could complete the tests and prove we didn't change much beyond naming convention. So I need this to confirm the gold standards are still being met.

Reply:
I think this is also a larger conversation than this PR since it'll require redoing the gold standards.

Since we're moving to the publication model, `Marks2017` wasn't published so it should be removed and since it has some issues with it still, it should probably be placed on it's own branch for later if we want to revisit.

Reply:
Ok. So hold off on deleting it until we make the adjustments to the gold files which will take place after our discussion next week?